src/test/spec/json/change-streams/legacy/change-streams-errors.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
we're not actually running any of the legacy ones right? i think we ought to just leave them out then to avoid any red-herring greps and what not.

Reply:
Good call, done.